cnf/Makegap.in

```diff
@@ -231,8 +231,7 @@ finfield.o: ../../Makefile @srcdir@/finfield.c @srcdir@/system.h config.h @srcdi
  @srcdir@/gvars.h @srcdir@/calls.h @srcdir@/opers.h @srcdir@/ariths.h \
  @srcdir@/bool.h @srcdir@/integer.h @srcdir@/finfield.h @srcdir@/records.h \
  @srcdir@/precord.h @srcdir@/lists.h @srcdir@/plist.h @srcdir@/string.h \
- @srcdir@/code.h @srcdir@/hpc/aobjects.h @srcdir@/hpc/thread.h @srcdir@/hpc/tls.h \
- @srcdir@/hpc/ffdata.h
+ @srcdir@/code.h @srcdir@/hpc/aobjects.h @srcdir@/hpc/thread.h @srcdir@/hpc/tls.h
 	$(CC) $(CPPFLAGS) $(GMP_CFLAGS) $(CFLAGS) -o finfield.o -c @srcdir@/finfield.c
```

Comment:
BTW, are you using `cd cnf && touch Makefile && make` to regenerate this file? I am just wondering, because when I do that, I get a file with a lot changes (due to different indentation).... Huh?!